README.md

```diff
@@ -4,47 +4,30 @@ This monorepo comprises a suite of tools **currently in development** to allow d
 
 ## Prerequisites
 
-Node v16+, NPM v7+
+Node v18+, NPM v7+
 
 ## Quick Start
 
 Clone this repo and run `npm i` from the root directory.  This project leverages [`npm workspaces`](https://docs.npmjs.com/cli/v7/using-npm/workspaces) so requires NPM v7 or above.
 
-- Start the cli - `npm run start-cli` and you should see some logs like below indicating the node is starting up
+- From the `packages/cli` subfolder, start the Ultralight `nodejs` client - `npm run dev` and you should see some logs like below indicating the node is starting up:
 ```sh
 de2f8:ultralight Started JSON RPC Server address=http://localhost:8545
-de2f8:discv5:service Updated ENR based on public multiaddr to enr:-IS4QH2xRY1ov...
 ```
-- Copy the node's ENR which should start with `enr:-IS...`
-  - Alternatively, retrieve the node's ENR using the JSON-RPC.  Details [here](./packages/cli/README.md)
-- Start the browser client dev server - `npm run start-browser-client` and open a browser window at `localhost:8080`
-- Click `Start Node`, paste the ENR into the input that says `Node ENR` and press `Connect To Node`
-- You should see a node appear in the table on the left side of the page
-- In the terminal where the Ultralight-CLI client is running, you should see something like below:
-```js
-de2f8:portalnetwork Received History Subnetwork request +25s
-de2f8:portalnetwork TALKREQUEST with PING message received from b81736575498a5850b0dd52f2695268cf60fe6c89ab74289692c5225c9e4e09e +0ms
-de2f8:portalnetwork adding b81736575498a5850b0dd52f2695268cf60fe6c89ab74289692c5225c9e4e09e 
-```
-- Try retrieving a block from the network.  (Hint: Enter `0x88e96d4537bea4d9c05d12549907b32561d3bf31f45aae734cdc119f13406cb6` - the hash for Block 1 from the Ethereum Mainnet in the input box on the right input and press the "Get Block by Blockhash")
+
+This will start a single instance of the Ultralight client running locally.  
 
 ## Connecting to the testnet
 
-1.  Follow the above quickstart guide as far as installing all dependencies.
-2.  From the repository root, run `node packages/proxy/dist/index.js --nat=extip`
-3.  Run `npm run start-browser-client`
-4.  Connect to one of the [Ultralight bootnodes](./packages//cli/bootnodes.txt)
-5.  You should start to see the network table populate with additional nodes as the client fills in its routing table
-6.  Try and get any block by blockhash from the Ethereum Mainnet between blocks 1-2500. 
-7.  **Crossing fingers** You should see the block header details before too long
+Follow the above quickstart guide and run `npm run dev-testnet` to start the client with the current list of bootnodes.  The client will attempt to ping all of bootnodes in the provided default bootnode list.
```

Comment:
Just running all the instructions.

Will post some observations here post merge.

Dev testnet command seems to work, here some extract from the output:

<img width="1102" alt="grafik" src="https://github.com/ethereumjs/ultralight/assets/931137/b8e27d96-5a2e-45c1-b209-f68114de1eba">

It would be nice to have an additional note what kind of network this is actually connecting to (in this case seems: History network) and also to have one addtional sentence on what is then actually happening (just the rough thing ("this is then downloading blocks from PN from mainnet" or whatever is happening).

Also for whole Portal Network, it would be generally good if testnets would have URLs, being it an issue on the Portal network GitHub repo or whatever. Atm these things are so much flurrying around in the sphere ("some" testnet "somewhere" which might or might not persist" 😅).

Doesn't need to be something big, but a few basic infos (when did the testnet start, some few parameters, how long should it run, stuff like that).

Reply:
🙂 

Ok, and after one has done this (running the dev command) one is a bit lost about "what to try next". 

Would be nice to have some additional sentence about where one could go from here and what would be a good way to approach. Or whatever makes sense.

Reply:
Update: ah, just seeing this right only now that you have put all/most of the new documentation in the `cli` folder/package. So that would then be just the missing link I guess, a reference from here to `CLI`, like `See [cli](LINK) package for further instructions on how to connect to various ultralight networks from the CL and information about the networks you can connect to". Or something like that.